### PR TITLE
fix: linter

### PR DESCRIPTION
**Pull request by zarazan**
## Description

- Fix the golangci linter
- Resolve all remaining linter issues minus the ignite depreciation Woz is going to fix

closes: [#22 ](https://github.com/ojo-network/ojo/issues/22)

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: app/export.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func (app *App) ExportAppStateAndValidators(
 // prepare for fresh start at zero height
 // NOTE zero height genesis is a temporary feature which will be deprecated
 //
-//	in favour of export at a block height
+//	in favor of export at a block height
```
Comment (adamewozniak) [inline app/export.go:53]: did the linter catch this? lol

**Review by adamewozniak — APPROVED** [review]
lgtm! let's merge https://github.com/ojo-network/ojo/pull/33 first so that the linter will be green

File: app/export.go (hunk 1 above)
Comment (zarazan) [reply app/export.go:53]: yep 😝

**Comment by adamewozniak** [comment]
@zarazan enabled only-squash-merges 😉

**Pull request merged**